tests/support/proxy.rs

```diff
@@ -273,7 +273,10 @@ fn run(proxy: Proxy, mut env: app::config::TestEnv) -> Listening {
     // printlns will show if the test fails...
     println!(
         "proxy running; destination={}, identity={:?}, inbound={}{}, outbound={}{}, metrics={}",
-        control_addr,
+        control_addr
+            .as_ref()
+            .map(|i| format!("{}", i))
```

Comment:
TIOLI: I think you could also use
```suggestion
            .map(SocketAddr::to_string)
```